### PR TITLE
#169571349 Improves test coverage and validate entry ID

### PR DESCRIPTION
**Pull request by Muhire-Josue**
#### What does this PR do?

- It improves the tests coverage and validates entry ID

#### Description of Task to be completed?

- Increase test coverage
- Handle error 500
- Split route file into userRoutes and entryRoutes
- Handle invalid ID from URL
- Validation test 

#### How should this be manually tested?

- Use this link:  https://mydiary-api-host.herokuapp.com/api/v1/entries/:entryId

#### Any background context you want to provide?

- N/A

#### What are the relevant pivotal tracker stories?

- [169571349](https://www.pivotaltracker.com/story/show/169571349)

#### Screenshots (if appropriate)
invalid-id

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,16 +1,24 @@
 /* eslint-disable no-console */
 import express from 'express';
-import routes from './server/routes/router';
+import userRoutes from './server/routes/userRouter';
```
Comment (ekumamait) [inline server.js]: You should import these two files into an `routes/index.js` file which you will then import here and use.

File: server/tests/validations/userValidation.test.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import chai from 'chai';
+import userSchema from '../../validations/userValidation';
+import mockData from '../data/mockData';
+
+const user = mockData[16];
+
+describe('validation test', () => {
+  it('Should validate', () => {
```
Comment (ekumamait) [inline server/tests/validations/userValidation.test.js]: the tests should be descriptive, all of them are called `should validate` but what is the test validating

**Comment by ekumamait** [comment]
The PR title is against the standard, please fix it and it does not have a story?

File: server.js [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
+import routes from './server/routes/index';
+
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 app.use(express.json());
-app.use(userRoutes);
-app.use(entryRoutes);
+app.use(routes);
 app.listen(PORT, () => console.log(`server running on port ${PORT}`));
```
Comment (ekumamait) [inline server.js]: we do not push development tools into production code, remove all the console.logs()

Comment (Muhire-Josue) [reply server.js]: I have removed all console.logs()

**Review by frankhn — APPROVED** [review]
LGTM.

**Pull request merged**